src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -2279,20 +2354,18 @@ TEST_CASE("contract storage", "[tx][soroban][archival]")
             client.has("key2", ContractDataDurability::PERSISTENT, false)));
     }
 
-    SECTION("read bytes limit enforced")
+    SECTION("live state does not meter read bytes")
     {
         // Write 5 KB entry.
         REQUIRE(isSuccess(client.resizeStorageAndExtend("key", 5, 1, 1)));
 
         auto readSpec =
             client.readKeySpec("key", ContractDataDurability::PERSISTENT)
```

Comment:
We should be able to set read bytes to 0 in p23 can't we?

Also, just a reminder from the comment above to try preserving p22 coverage.